NachoSpigot-Server/src/main/java/me/rastrian/dev/PlayerMap.java

```diff
@@ -0,0 +1,232 @@
+package me.rastrian.dev;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import com.google.common.base.Function;
+import com.google.common.base.Predicate;
```

Comment:
Any reason you're using Guava classes here as opposed to [java.util.function](https://docs.oracle.com/javase/8/docs/api/java/util/function/package-summary.html) ones?